1400-continuous-integration.md

```diff
@@ -5,8 +5,35 @@ aliases: [ "05-continuous-integration" ]
 
 ## With GitHub Actions 
 
-You tipi subscription can be used to run your build and test your project in GitHub actions.
+The tipi subscription can be used to run your build and test your project in GitHub actions.
 
 Run `tipi ci` to generate the YAML configuration for GitHub CI.
 
-See [Command Line authentication](04-authentication#authentication-in-continuous-integration-context) to handle authentication in CI workflows.
+### Authenticating on tipi.build from CI workflows
+Information are available as reference here : [Command Line authentication](13-environment-variables#command-line-authentication) to handle authentication in CI workflows.
```

Comment:
'Information' is a singular noun, and must take a singular verb => Information *is* available